doc/data/messages/n/nan-comparison/bad.py

```diff
@@ -0,0 +1,5 @@
+import numpy
+
+x = 42
+if x == numpy.NaN:  # [nan-comparison]
+    print("Bad!")
```

Comment:
```suggestion
def both_nan(x, y) -> bool:
   return x == np.NaN and y == float("nan")  # [nan-comparison]
```

Reply:
I think this suggestion adds unnecessary complexity and is harder to differentiate what the "problem" is

Reply:
if x = 42 then there's no need to compare it, so a function with unknown value is better imo, Then if we just return np.isnan(x) it's not a very good example, right, we could use isnan directly. Also the other variable permit to illustrate the two majors way to create a nan comparison.